tests/specs/concrete-rules.txt

```diff
@@ -25,7 +25,7 @@ EVM.Rsstore.old
 EVM-TYPES.#asByteStack
 EVM-TYPES.#asByteStackAux.recursive
 EVM-TYPES.#asWord.recursive
-EVM-TYPES.ByteArray.range
+EVM-TYPES.Bytes.range
 EVM-TYPES.bytesRange
 EVM-TYPES.mapWriteBytes.recursive
```

Comment:
Here in concrete-rules.txt we are making a change, that I don't see the same change happening in kevm.py? Are you sure we don't need to make the same change there?